auth/certificate_authenticator.cc

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright (C) 2022-present ScyllaDB
+ *
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#include "auth/certificate_authenticator.hh"
+
+#include <regex>
+
+#include "utils/class_registrator.hh"
+#include "replica/database.hh"
```

Comment:
Probably unnecessary to include this expensive replica-side header.

Reply:
Fixed.